app/v1/controllers/RetrieveBusinessDetailsController.scala

```diff
@@ -69,12 +65,6 @@ extends AuthorisedController(cc) with BaseController with Logging {
 
           val response = Json.toJson(vendorResponse)
 
-          auditSubmission(RetrieveBusinessDetailsAuditDetail(
-            userDetails = request.userDetails,
-            nino = nino,
-            `X-CorrelationId` = serviceResponse.correlationId,
-            AuditResponse(OK, Right(Some(response)))))
-
           Ok(Json.toJson(vendorResponse))
```

Comment:
```suggestion
          Ok(response)
```
or delete `val response = ...`